plugins/redis/redis.cpp

```diff
@@ -726,10 +729,17 @@ void Connection::handleLockOnGet(ICodeContext * ctx, const char * key, MemoryAtt
     if (lock(ctx, key, channel.str()))
         return;
 
+#if(0)
```

Comment:
minor: If you want to retain this code then it should have a comment why it would be useful/when you would use it.